imxrt-hal/Cargo.toml

```diff
@@ -45,4 +45,4 @@ rt = ["imxrt-ral/rt"]
 nosync = ["imxrt-ral/nosync"]
 
 [package.metadata.docs.rs]
-features = ["imxrt1062"]
```

Comment:
It's unclear why this change is here

Reply:
I don't either, by the looks of it a stray newline got deleted. My bad!